packages/plugin-ext/src/plugin/workspace.ts

```diff
@@ -64,6 +65,40 @@ export class WorkspaceExtImpl implements WorkspaceExt {
         });
     }
 
+    findFiles(include: GlobPattern, exclude?: GlobPattern | undefined, maxResults?: number,
+              token: CancellationToken = CancellationToken.None): PromiseLike<Uri[]> {
+        let includePattern: string;
+        if (include) {
+            if (typeof include === 'string') {
+                includePattern = include;
+            } else {
+                includePattern = include.pattern;
+            }
+        } else {
+            includePattern = '';
+        }
+
+        let excludePatternOrDisregardExcludes: string | false;
+        if (exclude === null) {
```

Comment:
do we have somtimes null ? AFAIK in theia we mostly have undefined or values

Reply:
Yes, we should not use `null` at all

Reply:
fixed